gapii/cc/spy.cpp

```diff
@@ -278,7 +225,7 @@ Spy::~Spy() {
   endTraceIfRequested();
 }
 
-void Spy::resolveImports() { GlesSpy::mImports.resolve(); }
+void Spy::resolveImports() {}
```

Comment:
This function is no longer called. Delete it.

Reply:
Done.